codalab/model/bundle_model.py

```diff
@@ -529,7 +529,10 @@ def make_condition(key, field, value):
                         alias(select([cl_worksheet_item.c.bundle_uuid]).where(condition))
                     )
             elif key in ('.before', '.after'):
-                target_datetime = parser.isoparse(value)
+                try:
+                    target_datetime = parser.isoparse(value)
+                except Exception:
```

Comment:
more specific exception

Reply:
Changed to `ValueError` which is thrown by `isoparse`.